docs/elements.md

```diff
@@ -58,7 +58,7 @@ images, files, and code display. The following **decorative** elements are avail
 - [`pl-file-download`](#pl-file-download-element): Enable file downloads for
   data-centric questions.
 - [`pl-file-preview`](#pl-file-preview-element): Displays a preview of submitted files.
-- [`pl-graph`](#pl-graph-element): Displays graphs, using GraphViz DOT notation, an adjacency matrix, or a networkx graph.
+- [`pl-graph`](#pl-graph-element): Displays graphs, using GraphViz DOT notation, an adjacency matrix, or a network graph.
```

Comment:
This was actually correct, [networkx is a library](https://networkx.org/).

Reply:
```suggestion
- [`pl-graph`](#pl-graph-element): Displays graphs, using GraphViz DOT notation, an adjacency matrix, or a [`networkx`](https://networkx.org/) graph.
```

Reply:
@nwalters512 if there's a link to networkx here, should there be a link to [GraphViz DOT notation](https://graphviz.org/doc/info/lang.html) as well?

Reply:
I'd be fine with that (though IMO those docs aren't very user-friendly).

Reply:
Is it ok as is with the new change? Or should it be [`GraphViz DOT notation`](https://graphviz.org/doc/info/lang.html) instead of [GraphViz DOT notation](https://graphviz.org/doc/info/lang.html).

Reply:
This is good, thanks.